ksqldb-serde/src/main/java/io/confluent/ksql/serde/avro/AvroSchemaTranslator.java

```diff
@@ -57,8 +58,8 @@ public Schema toConnectSchema(final ParsedSchema schema) {
 
   @Override
   public ParsedSchema fromConnectSchema(final Schema schema) {
-    final Schema avroCompatibleSchema = AvroSchemas
-        .getAvroCompatibleConnectSchema(schema, formatProps.getFullSchemaName());
+    final Schema avroCompatibleSchema = SchemaFullNameAppender
```

Comment:
Do we not need this change for JSON_SR as well? Does JSON_SR not have the same schema name requirements?